presidio-anonymizer/app.py

```diff
@@ -25,9 +28,11 @@ def anonymize():
                 data = AnonymizerRequest(content)
                 text = AnonymizerEngine().anonymize(data)
             except InvalidParamException as e:
-                return e.err, 400
-            except Exception:
-                # TODO add logger 2652
+                logging.warning(
+                    {f"failed to anonymize text with validation error: {e.err_msg}"})
+                return e.err_msg, 422
```

Comment:
Good change to add api tests to the functional tests module to verify that error.

Reply:
Added.

Reply:
I meant to the functional tests module

Reply:
yes, I know, didn't push it yet but it's added.